### ISSUE TITLE
Data inconsistencies

### ISSUE THREAD

**Issue by yatil** [w3c/wai-wcag-quickref#26]
Please use this issue to report inconsistencies with the data structure or display of the data, for example if Success Critera are missing techniques or the techniques are displayed in the wrong way. Use the following Template to report (Just note the SC number and the Techniques. Use a line for every criterion but you can mention multiple techniques per line):

``` markdown
* **8.9.9** Advisory missing
* **9.9.9** SL99, C929 missing
* …
```

**Comment by yatil** [comment]
- **1.1.1** Advisory missing

**Comment by bakkenb** [comment]
Additional (Advisory) Techniques and Common Failures don't seem to be displayed correctly. I was looking through 1.2.3 through 1.2.6 and they are not matching the actual understandings pages. 

Not sure if you were still working on this or if it was a bug. I didn't check them further.

**Comment by HerinHentry** [comment]
**1.1.1** Situation B _ missing short text alternative techniques & long text alternative techniques.

**Comment by iadawn** [comment]
- **3.3.4** F82 incorrectly associated

**Comment by yatil** [comment]
@iadawn 

> 3.3.4 F82 incorrectly associated

This was fixed with #110.

**Comment by yatil** [comment]
FF! Finally fixed.
